deepchecks/checks/performance/class_performance_imbalance.py

```diff
@@ -0,0 +1,240 @@
+"""The class performance imbalance check."""
+#pylint: disable=inconsistent-quotes
+
+import typing as t
+from collections import defaultdict
+
+import pandas as pd
+import numpy as np
+from sklearn.metrics import precision_score, recall_score, get_scorer, make_scorer
+from sklearn.utils.multiclass import unique_labels as get_unique_labels
+
+from deepchecks import SingleDatasetBaseCheck, Dataset, CheckResult, ConditionResult
+from deepchecks.utils.metrics import task_type_validation, ModelType
+from deepchecks.utils.strings import format_percent
+from deepchecks.errors import DeepchecksValueError
+
+
+__all__ = ['ClassPerformanceImbalanceCheck']
+
+
+ScorerFunc = t.Callable[
+    [object, pd.DataFrame, pd.Series], # model, features, labels
+    np.ndarray # scores
+]
+
+AlternativeScorer = t.Union[str, ScorerFunc]
+
+
+CP = t.TypeVar('CP', bound='ClassPerformanceImbalanceCheck')
+
+
+class ClassPerformanceImbalanceCheck(SingleDatasetBaseCheck):
```

Comment:
In other checks we don't use the word `check` in the class name, so better to have the same

Reply:
oh, you are right, other check types do not use 'Check' noun in the name, renamed